shared-signals/template.yaml

```
@@ -112,3 +108,37 @@ Resources:
       Name: SSFUserPoolArn
       Type: String
       Value: !GetAtt SSFUserPool.Arn
+
+  CognitoDomainURLParameter:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Name: CognitoDomainUrl
+      Type: String
+      Value: !Sub https://di-txma-${AWS::StackName}-${Environment}-ssf-auth.auth.eu-west-2.amazoncognito.com
```

```suggestion
      Value: !Ref SSFUserPoolDomain
```
Ref returns the user pool domain, so no need to type it out.

---

did a quick check, 
```yaml
Outputs:
  SSFUserPoolId:
    Description: 'user pool id for cognito'
    Value: !Ref SSFUserPoolDomain
```

gives me 
```bash
CloudFormation outputs from deployed stack
---------------------------------------------------------------------------------------------------------------------------------------------------
Outputs                                                                                                                                           
---------------------------------------------------------------------------------------------------------------------------------------------------
Key                 SSFUserPoolId                                                                                                                 
Description         user pool id for cognito                                                                                                      
Value               di-txma-shared-signals-infra-dev-ssf-auth                                                                                     

Key                 UserConfigTableName                                                                                                           
Description         table name for user config table                                                                                              
Value               shared-signals-infra-user-config                                                                                              
---------------------------------------------------------------------------------------------------------------------------------------------------


Successfully created/updated stack - shared-signals-infra in eu-west-2
```

so i could do 

```suggestion
      Value: !Sub https://${!Ref SSFUserPoolDomain}.auth.eu-west-2.amazoncognito.com
```

thoughts?

---

integration tests work with updated url